app/controllers/facility_reservations_controller.rb

```diff
@@ -178,8 +182,12 @@ def destroy
   def admin_reservation_params
     admin_params = params.require(:admin_reservation).permit(:admin_note,
                                                              :expires_mins_before,
-                                                             :category)
+                                                             :category,
+                                                             :repeat_frequency,
+                                                             :repeat_end_date)
     admin_params[:expires_mins_before] = nil if params[:admin_reservation][:expires] == "0"
+    admin_params[:repeat_frequency] = nil if params[:admin_reservation][:repeats] == "0"
```

Comment:
I feel like now that we're using a form object, these things could be pulled into the form.